CHANGELOG.md

```diff
@@ -0,0 +1,260 @@
+## 1.0.0-beta.1 (2019-05-31)
+
+* feat(build): build pipeline add ([482c466](https://github.com/eva-design/eva/commit/482c466))
```

Comment:
there are two versions in the changelog with repeating commits, remove 0.0.1 version and leave 1.0.0-beta.1 only